server/channels/api4/custom_profile_attributes.go

```diff
@@ -25,6 +25,11 @@ func (api *API) InitCustomProfileAttributes() {
 }
 
 func listCPAFields(c *Context, w http.ResponseWriter, r *http.Request) {
+	if c.App.Channels().License() == nil {
```

Comment:
I guess this needs to be a check against enterprise SKUs https://hub.mattermost.com/abc/pl/cwgo68naxbdgzkiug6mwexuupe

Reply:
Updated to check for Enterprise licenses, PTAL https://github.com/mattermost/mattermost/pull/29906/commits/7dabf656b370f25ed3f07550e4e1373949514ff1

Reply:
Thanks, @mgdelacroix!